src/Avalonia.Visuals/Rendering/SceneGraph/SceneBuilder.cs

```diff
@@ -201,6 +205,7 @@ private static void Update(DrawingContext context, Scene scene, VisualNode node,
                     node.ClipBounds = clipBounds;
```

Comment:
@grokys there are some non-trivial calculations that are applied to `clipBounds` here, so I'm not sure how well my change would work with transforms and such.

Reply:
Looks good to me and just tested it in the `RenderDemo` with a rotation transform animation: all looks to me good.